File: app/src/main/java/com/example/rostrzep/googlebooksapp/api/GoogleBooksService.java
package com.example.rostrzep.googlebooksapp.api;

import com.example.rostrzep.googlebooksapp.api.model.BooksList;

import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Observable;

public interface GoogleBooksService {

    @GET("volumes")
    Observable<BooksList> searchForBooks(@Query("q") final String searchQuery, @Query("maxResults") final Integer maxResults);
}

File: app/src/main/java/com/example/rostrzep/googlebooksapp/api/GoogleBooksService.kt
package com.example.rostrzep.googlebooksapp.api

import com.example.rostrzep.googlebooksapp.api.model.BooksList

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface GoogleBooksService {

    @GET("volumes")
    fun searchForBooks(@Query("q") searchQuery: String, @Query("maxResults") maxResults: Int?): Observable<BooksList>
}